scipy/stats/_stats_py.py

```diff
@@ -10315,7 +10320,18 @@ def _prk(r, k):
     return (-1)**(r-k)*special.binom(r, k)*special.binom(r+k, k)
 
 
-@xp_capabilities(np_only=True)
+def _triu(x, xp):
+    # not array API compatible, but supported by cupy, jax.numpy, torch, dask.array
```

Comment:
Maybe my brain is still asleep but `triu` is in the standard?

Reply:
Ah : ) So I was asleep when writing this. I thought it was, too... not sure how I didn't find it when I was looking. I'll fix that!